.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
           - "--maxlevel"
           - "2"
   - repo: https://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.3.1
+    rev: v1.5.4
```

Comment:
@ambika-garg I don't think these are relevant changes to what you are trying to do. Can you please remove these?